content/docs/next-release/operator.md

```diff
@@ -859,7 +859,7 @@ spec:
     options:
       default_strategy:
         type: probabilistic
-        param: 50
+        param: .5
```

Comment:
I am not able to parse this form:
```
failed to unmarshal strategies: invalid character '.' looking for beginning of value
```

Please update to `0.5`

Reply:
Done!